### PR TITLE
Cleaned the UI a little

### PR DESCRIPTION
**Pull request by mendykrauss**
I updated the calendar list template to be a little more user friendly. (I think)

### CODE REVIEW

File: my_hebrew_dates/templates/hebcal/calendar_list.html [hunk 1]
```diff
@@ -3,44 +3,57 @@
...
+              <th class="text-center">Name</th>
+              <th class="text-center">Time Zone</th>
+              <th class="text-center"># of Events</th>
+              <th class="text-center">Actions</th>
+            </tr>
+          </thead>
+          <tbody>
+            {% for calendar in calendar_list %}
+            <tr>
+              <td class="py-3"><a class="text-decoration-none fs-6" href="{% url 'hebcal:calendar_share' calendar.uuid %}">B-day</a></td>
```
Comment (abe-101) [inline my_hebrew_dates/templates/hebcal/calendar_list.html:29]: Why `B-day`?
Do you mean the name of the calendar?
That can be accessed by `calendar.name`

**Review by abe-101 — CHANGES_REQUESTED** [review]
Thank you for PR
I do like how it looks
Can you make the small request change and I'll merge?

File: my_hebrew_dates/templates/hebcal/calendar_list.html [hunk 2]
```diff
@@ -3,44 +3,57 @@
...
+              <td class="py-3 text-muted">America/New_York</td>
+              <td class="py-3">1</td>
```
Comment (abe-101) [inline my_hebrew_dates/templates/hebcal/calendar_list.html:31]: we need to dynamicly get the number of events

File: my_hebrew_dates/templates/hebcal/calendar_list.html (hunk 1 above)
Comment (mendykrauss) [reply my_hebrew_dates/templates/hebcal/calendar_list.html]: I committed the fix

**Comment by abe-101** [comment]
Thanks for the update
I've gone ahead and fixed the event count

**Review by abe-101 — APPROVED** [review]

**Pull request merged**